src/tcas/src/components/TcasComputer.ts

```diff
@@ -9,7 +9,6 @@ import { TcasComponent } from '@tcas/lib/TcasComponent';
 import { LatLongData } from '@typings/fs-base-ui/html_ui/JS/Types';
 import { LocalSimVar } from '@shared/simvar';
 import { NXDataStore } from '@shared/persistence';
-import stringify from 'safe-stable-stringify';
```

Comment:
This was a performance optimisation, as the Coherent JSON.stringify is very slow.

Reply:
It seems like that the `JSON.stringify` is called in `ViewListener.triggerToAllSubscribers`. If you know that this implementation is more optimized, then to avoid double serialization calls of `triggerToAllSubscribers` can be replaced with
```
Coherent.trigger('TRIGGER_EVENT_TO_ALL_SUBSCRIBERS", this.sendListener.m_name, 'A32NX_TCAS_TRAFFIC', stringify(this.sendAirTraffic))
```

`JSON.stringify` is likely implemented in C, so I wonder how the JS implementation of stringify is better. It must cut some corners. Just curious...